### PR TITLE
docs: LSDV-5553: Add styling for info/tip admonition

### PR DESCRIPTION
**Pull request by caitlinwheeless**
We currently have several note types - note (gray), warning (orange), attention (red). Added styling for an info note that can be used for tips or ideas:

```
!!! info tip
     You can do XYZ to reduce your time spent doing ABC!
```

### CODE REVIEW

File: docs/themes/v2/source/css/styles.css [hunk 1]
```diff
@@ -1085,6 +1086,13 @@ main {
   --color-background: var(--color-yellow-5);
 }
 
+.content-markdown .admonition.info {
+  --color-border: #DD7DFF;
```
Comment (huguestennier) [inline docs/themes/v2/source/css/styles.css]:
```suggestion
  --color-border: var(--color-purple-90);
```

File: docs/themes/v2/source/css/styles.css [hunk 2]
```diff
@@ -1085,6 +1086,13 @@ main {
...
+  --color-title: var(--color-white);
+  --color-title-background: #DD7DFF;
```
Comment (huguestennier) [inline docs/themes/v2/source/css/styles.css]:
```suggestion
  --color-title-background: var(--color-purple-90);
```

File: docs/themes/v2/source/css/styles.css [hunk 3]
```diff
@@ -1085,6 +1087,13 @@ main {
...
+  --color-background: #FDF6FF;
```
Comment (huguestennier) [inline docs/themes/v2/source/css/styles.css]: Could we reuse the Purple (Plum) 0 here? The shade is slightly different `#FBF2FC` but it should work

```suggestion
  --color-background: var(--color-purple-0);
```

Comment (caitlinwheeless) [reply docs/themes/v2/source/css/styles.css]: Sounds good

**Review by huguestennier — APPROVED** [review]

**Pull request merged**